components/style/stylesheets.rs

```diff
@@ -99,6 +103,64 @@ pub struct Namespaces {
     pub prefixes: FnvHashMap<Prefix , Namespace>,
 }
 
+/// Like gecko_bindings::structs::MallocSizeOf, but without the Option<> wrapper. Note that
+/// functions of this type should not be called via do_malloc_size_of(), rather than directly.
+#[cfg(feature = "gecko")]
+pub type MallocSizeOfFn = unsafe extern "C" fn(ptr: *const c_void) -> usize;
+
+/// Call malloc_size_of on ptr, first checking that the allocation isn't empty.
+#[cfg(feature = "gecko")]
```

Comment:
I'd prefer to not compile it out if possible, even if it's not used, because if someone is changing related code and causes this not to compile would only see the error when compiling the geckolib version.